reference/channels.v3.yml

```diff
@@ -3505,6 +3505,12 @@ components:
           - currencies
           - notifications
           - localization
+          - overview
```

Comment:
@bc-andreadao can we list them in the order they appear in CP UI?
<img width="230" alt="Screenshot 2024-09-12 at 12 12 34" src="https://github.com/user-attachments/assets/690f91fa-636f-4079-be3d-42475fc9627d">

Reply:
Sounds good, just reordered them